lib/amazon_mws.rb

```diff
@@ -126,6 +126,13 @@ def close(ref_number)
       })
   end
 
+  def cancel(ref_number)
+    process({
+      "Action" => "CancelOrderReference",
+      "AmazonOrderReferenceId" => ref_number
+      })
+  end
+
```

Comment:
It's generally easier to write isolated tests with dependency stubbing than writing all-through integration sort of tests. Rspec protects you very well on that front with the _verification of doubles_, so there's no danger of stubbing a method that someone has later deleted.